docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -44,19 +44,29 @@ Set to complete URL to override Apple/iOS icon.
 
 List the link IDs to use here. Grafana will look for matching link configurations, the link IDs should be space-separated and contain no whitespace.
 
-## [meta_analytics]
+## [usage_insights.export]
 
-### max_file_age
+### enabled
+
+Enable the usage insights export feature.
+
+### storage
+
+Storage type. Example: `loki`.
```

Comment:
Currently, the only possiblity is `loki`. It's a bit weird to ask for a storage type now but it's because we're planning to add more.

Reply:
Ah! OK. Is it possible to no include the possibility to add more atm or is that too much of a code change? Yes, seems weird, yet development rationale could certainly outweigh the weirdness.

Reply:
@AgnesToulet could we make `loki` the _default_ setting? It is indeed weird to include the setting if there is only one option, though it is obvious why you designed it that way. We could strike a middle ground by defaulting to the only option for now, so that the user doesn't need to do anything.

Reply:
It's already the default setting. I made an update to the documentation, tell me if it's not enough.